include/nbl/video/utilities/IPropertyPool.h

```diff
@@ -34,19 +38,19 @@ class NBL_API2 IPropertyPool : public core::IReferenceCounted
         inline bool isContiguous() const {return m_indexToAddr;}
 
         //
-        inline uint32_t getAllocated() const
+        inline uint64_t getAllocated() const
         {
             return indexAllocator.get_allocated_size();
         }
 
         //
-        inline uint32_t getFree() const
+        inline uint64_t getFree() const
         {
             return indexAllocator.get_free_size();
         }
 
         //
-        inline uint32_t getCapacity() const
+        inline uint64_t getCapacity() const
```

Comment:
no point changing them to `uint64_t` if your address allocator is still  `uint32_t`

Reply:
also this is a poster child fro `using` aliasing the `PropertyAddressAllocator::value_type` and using that throughout